### ISSUE TITLE
shift-drag-dropping links from a browser into mpv adds blank playlist entries

### ISSUE THREAD

**Issue by jgreco** [mpv-player/mpv#6241]
### mpv version and platform
mpv 0.29 on OSX

### Reproduction steps
Hold down the shift key and drag a youtube video link from a web browser into mpv.

### Expected behavior
The link is appended to the end of mpv's playlist.

### Actual behavior
The link is appended to the end of mpv's playlist, but an additional blank playlist entry is also created.
![screen shot 2018-10-24 at 12 05 28 pm](https://user-images.githubusercontent.com/41033/47457351-5f780e00-d7c6-11e8-8edf-19699b3c032c.png)

### Log file
```
[  15.233][d][cplayer] Run command: loadfile, flags=1, args=[https://www.youtube.com/watch?v=MVuw2CIVtKg, append-play, ]
[  15.233][d][cplayer] Run command: loadfile, flags=1, args=[, append-play, ]
```

**Comment by sfan5** [comment]
related? #6229

**Comment by Akemi** [comment]
i just tested it, and it didn't fix it.

**Comment by Akemi** [comment]
fixed with #6246.
